client/packages/requisitions/src/ResponseRequisition/DetailView/Tabs/Documents.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { ReactElement, useMemo } from 'react';
+import { Grid } from '@openmsupply-client/common';
+import { ResponseRowFragment } from '../../api';
+import { DocumentsTable } from '@openmsupply-client/system';
+
+interface DocumentsProps {
+  data?: ResponseRowFragment;
+  invalidateQueries: () => Promise<void>;
+}
+
+export const Documents = ({
+  data,
+  invalidateQueries,
+}: DocumentsProps): ReactElement => {
+  const deletableDocumentIds = useMemo(() => {
+    const requisitionId = data?.id ?? '';
+    return new Set(
+      (data?.documents.nodes ?? [])
+        .filter(doc => doc.recordId === requisitionId)
+        .map(doc => doc.id)
+    );
+  }, [data]);
+
+  return (
+    <>
+      <Grid flex={1} display="flex" flexDirection="column">
+        <Grid flex={1} sx={{ boxShadow: theme => theme.shadows[2] }}>
+          <DocumentsTable
+            recordId={data?.id ?? ''}
+            documents={data?.documents.nodes ?? []}
+            tableName="requisition"
+            invalidateQueries={invalidateQueries}
+            deletableDocumentIds={deletableDocumentIds}
+          />
+        </Grid>
+      </Grid>
+    </>
+  );
```

Comment:
You need the Grid container, nor the "fragment" markers `<> ,</>`.

Reply:
422c1d37d6